lib/asciidoctor/converter/docbook5.rb

```diff
@@ -374,6 +374,14 @@ def table node
       result = []
       pgwide_attribute = (node.option? 'pgwide') ? ' pgwide="1"' : nil
       result << %(<#{tag_name = node.title? ? 'table' : 'informaltable'}#{common_attributes node.id, node.role, node.reftext}#{pgwide_attribute} frame="#{node.attr 'frame', 'all'}" rowsep="#{['none', 'cols'].include?(node.attr 'grid') ? 0 : 1}" colsep="#{['none', 'rows'].include?(node.attr 'grid') ? 0 : 1}">)
+
+      if (node.option? 'breakable')
+        result << %(<?dbfo keep-together="auto"?>)
```

Comment:
Can you switch to using single quotes around this string? The `%()` is only needed when using interpolation. Same for the other dbfo string.